airgun/entities/host.py

```diff
@@ -127,6 +127,11 @@ def delete(self, entity_name, cancel=False):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def select_checkbox(self):
```

Comment:
```suggestion
    def select_all_hosts(self):
```
I would suggest renaming this function so it includes the word `all` in it so that it describes the behavior more correctly.

Reply:
Can you try using play_ansible_roles here instead to select and run roles?
https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/host.py#L309C9-L309C27